src/main/java/nb/kafka/operator/KafkaUtilities.java

```diff
@@ -228,16 +231,22 @@ private void addAcl(Collection<AclBinding> acls, Collection<AclBinding> effectiv
   }
 
   TopicDescription topicDescription(String topicName) {
+    count = 0;
```

Comment:
Any particular reason why this is on object level and not local variable?

Reply:
i think that could be used in all methodes so i declared the MaxTries global static and  i init count in each function that we should retry

Reply:
I'd make each count local, as then JVM can optimize it easier.

Reply:
ok the issue still exits  
![image](https://user-images.githubusercontent.com/7292030/58487036-c9211780-811b-11e9-89a4-1faf53f042f7.png)

Reply:
The issue persist
![image](https://user-images.githubusercontent.com/7292030/58488176-c1fb0900-811d-11e9-9830-0fec49be8c5e.png)

Reply:
Can you check what has been setup on kafka side? This message means that we were unable to authorize, and I guess by default ENABLE_ACL is set to false in operator so it doesn't send credentials, and seems kafka deployment requires it. Can you see if it is the case, and disable acls on kafka side?

Reply:
ok i see if acl is enabled in the broker we get this issue  maybe we need to  log something  if this exception is occured  ?